flashinfer/grouped_mm/kernels/cutedsl/sm120_moe/moe_gemm_fp8.py

```
@@ -0,0 +1,614 @@
+                else:
+                    store_wg(acc, thr, thr_st, sC, st_atom, gD, tile, (bm, bn), N,
+                             cfg.store.out_dtype, store_empty, cfg.epi_bar_id, cfg.mma_threads)
+                tile = cons.get_next_tile(sWork, sfull, sempty)
+
+
+def _stream():
+    return cuda.CUstream(torch.cuda.current_stream().cuda_stream)
+
+
+def make_args(a_q, a_scale, b_q, b_scale, out, m_indptr):
+    sf = lambda t: from_dlpack(t.contiguous(), assumed_align=16).mark_layout_dynamic()
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `sf` as a `def`

(E731)